test/validators_test.exs

```diff
@@ -0,0 +1,137 @@
+defmodule ValidatorsTest do
+  @moduledoc false
+  use ExUnit.Case, async: true
+  use PropCheck
+
+  doctest Validatex.Validators
```

Comment:
Je v `Validatex.Validators` vubec nejaky doctest?

Reply:
Myslim, ze  neni, Oki smaznu to